include/cudaq/Optimizer/Transforms/Passes.td

```diff
@@ -172,6 +172,19 @@ def EraseNopCalls : Pass<"erase-nop-calls", "mlir::func::FuncOp"> {
   }];
 }
 
+def ExpandControlVeqs: Pass<"expand-control-veqs", "mlir::func::FuncOp"> {
+  let summary = "Expands Veqs (with statically known sizes) into individual qubits when they are used as controls.";
```

Comment:
nit: can we make this shorter still? (it would be nice to make it terse and fit in 80 columns.

Reply:
Done